web/l10n/en.json

```diff
@@ -359,6 +359,7 @@
     "notificationWeb": "Send via Web",
```

Comment:
What do you think about renaming these to just "web", "email" and "SMS"?

Reply:
OK. Then we can use `"notificator"` as prefix...

Reply:
Sure.